storage/migration/sstables.go

```diff
@@ -137,22 +137,28 @@ func writerSSTableWorker(ctx context.Context, workerIndex int, db *pebble.DB, ss
 	}
 }
 func createSSTableWriter(filePath string) (*sstable.Writer, error) {
-	f, err := vfs.Default.Create(filePath)
+	f, err := vfs.Default.Create(filePath, vfs.WriteCategoryUnspecified)
 	if err != nil {
 		return nil, err
 	}
 
 	writable := objstorageprovider.NewFileWritable(f)
 	sstWriter := sstable.NewWriter(writable, sstable.WriterOptions{
-		TableFormat: sstable.TableFormatMax,
+		// pebble 1 is using TableFormatPebblev4, pebble 2's latest is TableFormatPebblev5 (TableFormatMax)
+		// in order to be compatible with pebble 1, we use TableFormatPebblev4 for now.
+		// TODO: use TableFormatMax in next spork
+		// TableFormat: sstable.TableFormatMax,
+		TableFormat: sstable.TableFormatPebblev4,
 	})
 
 	return sstWriter, nil
 }
 
 func ForceCompactPebbleDB(pebbleDir string) error {
 	pebbleDB, err := pebble.Open(pebbleDir, &pebble.Options{
-		FormatMajorVersion: pebble.FormatNewest,
+		// TODO: use FormatNewest in next spork
+		// FormatMajorVersion: pebble.FormatNewest,
+		FormatMajorVersion: pebble.FormatVirtualSSTables,
```

Comment:
Ensure after switching to pebble2, we can still switch back to pebble1 in case needed.